src/elements/paginator/paginator.scss

```diff
@@ -146,7 +147,32 @@
     }
   }
 
-  &:hover {
+  &:disabled {
+    --sbb-paginator-page-color: var(--sbb-color-smoke);
+    --sbb-paginator-page-border-width: var(--sbb-border-width-1x);
+    --sbb-paginator-page-border-style: dashed;
+    --sbb-paginator-page-border-color: var(--sbb-color-smoke);
+    --sbb-paginator-page-background-color: var(--sbb-color-milk);
+    --sbb-paginator-page-cursor: default;
+
+    &[data-selected] {
+      --sbb-paginator-page-border-width: var(--sbb-border-width-2x);
+    }
+
+    @include sbb.if-forced-colors {
+      --sbb-paginator-page-border-color: GrayText;
+      --sbb-paginator-page-color: GrayText;
+      --sbb-paginator-page-background-color: Canvas;
+    }
+
+    :host([negative]) & {
```

Comment:
can you move negative before forced-colors?

Reply:
Done!